app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabFeature.kt

```diff
@@ -129,6 +148,12 @@ class RecentSyncedTabFeature(
     }
 }
 
+/**
+ * The number of days to search history for a preview image URL to display for a synced
+ * tab.
+ */
+private const val DAYS_HISTORY_FOR_PREVIEW_IMAGE = 3L
```

Comment:
I am not sure if 3 days might be too short. We have used 10 days in https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/components/bookmarks/BookmarksUseCase.kt#107.

Reply:
I used 3 here to try and keep it a little more performant, we're already struggling with the performance of this feature a little bit. My reasoning being that typically a "recent synced tab" has probably been visited on its original device within the last 3 days

Reply:
Thanks, sounds good to me.